R/helpFunsBayesian.R

```diff
@@ -173,9 +173,10 @@
 }
 
 
-.LRblav <- function(data, cmat, basell) {
+.LRblav <- function(data, cmat, basell, callback = function(){}) {
```

Comment:
Well if you do it for 10000 covariance matrices or more, it does take a few seconds at least

Reply:
hmmm well if you think it'd be worthwhile I'd be happy to try to make this a bit faster

https://github.com/jasp-stats/jaspReliability/blob/1bc7dc1d5bcf04f4010f19b8823dca3cf38ce3f0/R/helpFunsBayesian.R#L191-L203

there are multiple things that can be improved. In addition, this would probably benefit if it's vectorized over `varcov` right? What are the dimensions of `implieds`? (no items, no items, MCMC iter)?

Reply:
the dimensions are `chains x iter x items x items`. If you would like to make this faster be my guest :)